### PR TITLE
Update miniconda ci

### PR DESCRIPTION
**Pull request by richardhuang11**

### CODE REVIEW

File: ci/install.sh [hunk 1]
```diff
@@ -3,19 +3,22 @@
 set -e
 
 if [[ "$(uname)" == "Darwin" ]]; then
-    URL="https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.2-MacOSX-x86_64.sh"
+    URL="https://repo.anaconda.com/miniconda/Miniconda3-py37_4.10.3-MacOSX-x86_64.sh"
```
Comment (richardhuang11) [inline ci/install.sh:6]: This resolved the libffi 0.6 issue

File: ci/script.sh [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+export PATH="$HOME/.cargo/bin:$PATH"
+
+eval `ssh-agent -s`
+ssh-add
+
+CARGO_NET_GIT_FETCH_WITH_CLI=true
```
Comment (richardhuang11) [inline ci/script.sh:11]: Workaround to rust ssh authentication... 
```
Caused by:
  failed to authenticate when downloading repository: ssh://git@github.com/rust-lang/crates.io-index

  * attempted ssh-agent authentication, but no usernames succeeded: `git`
 ```

https://sathia27.github.io/posts/2021/08/19/rust-cargo-resolve-authentication-issue.html
https://doc.rust-lang.org/nightly/cargo/appendix/git-authentication.html#git-authentication

File: ci/script.sh [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+export PATH="$HOME/.cargo/bin:$PATH"
```
Comment (richardhuang11) [inline ci/script.sh:6]: Resolves `./ci/script.sh: line 5: cargo: command not found`

File: ci/script.sh (hunk 2 above)
Comment (richardhuang11) [reply ci/script.sh:11]: However `ci/script.sh` worked without any of this additional ssh config when I sshed into circleci?

Comment (richardhuang11) [reply ci/script.sh:11]: Getting the error once again on a new commit?

**Review by mrterry — APPROVED** [review]

**Comment by mrterry** [comment]
maybe remove the travis config too?

**Pull request merged**